### DISCUSSION TITLE
Add a setting that toggles stacking unstackable items on or off

### DISCUSSION THREAD

**Discussion by 12bowl** [QuickShop-Community/QuickShop-Hikari#1335] [Ideas]
Through usage of this plugin on a towny server, I realized that you could create stacks of unstackable items like swords, armor, ender pearls and potions through usage of the /qs size command.

Would it be possible to make the stacking of unstackable items toggleable? It would allow users to benefit from the shop plugin all the while ensuring the server stays mostly a vanilla experience.

![gagaga](https://github.com/Ghost-chu/QuickShop-Hikari/assets/153016813/b14c5b0d-e113-4bf4-984f-be0ef5a0b0f1)

**Comment by YuanYuanOwO** [comment]
allow-stacks is disable for default,please check your config.yml  
After a simple test, I can't reproduce the issue

**Comment by Ghost-chu** [comment]
cannot reproduce, check and restore `custom-item-stacksize` to default in config.yml
